contrib/kube-prometheus/jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```diff
@@ -4,6 +4,22 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
   _config+:: {
     namespace: 'default',
 
+    kubeStateMetrics+:: {
+      // when this is an empty string, you get the default set
+      collectors: '',
+      scrapeTimeout: '',
+      scrapeInterval: '30s',
+    },
+
+    resizer+:: {
```

Comment:
I think it would be more natural if we fold this into the `kubeStateMetrics` configuration.

Reply:
This was the bit I was stylistically unsure on. I bias towards `kubeStateMetrics` as well, but I followed the versions/imageRepos pattern. I'll push an update moving it!